certs.sh

```diff
@@ -21,6 +21,14 @@ openssl req -x509 -sha256 -nodes -days 10 -key "$CERTDIR"/ca_0.key \
   -extensions v3_ca \
   2> /dev/null
 
+# Inflate certificate for the amplification test
+fakedns=""
+if [ "$CHAINLEN" != "1" ]; then
+  for i in $(seq 1 20); do
+    fakedns="$fakedns,DNS:$(tr -dc 0-9A-Za-z < /dev/urandom | head -c 250)"
```

Comment:
This command doesn't work on MacOS:

```
$ tr -dc 0-9A-Za-z < /dev/urandom | head -c 250
tr: Illegal byte sequence
```

Adding `LC_CTYPE=C` env variable fixes it

```suggestion
    fakedns="$fakedns,DNS:$(LC_CTYPE=C tr -dc 0-9A-Za-z < /dev/urandom | head -c 250)"
```

Reply:
Applied now.  Thanks.